src/app/components/AccountAvatar/addressToJazzIconSeed.tsx

```diff
@@ -0,0 +1,20 @@
+import { staking } from '@oasisprotocol/client'
+
+const jsNumberForAddress = (address_eth: string) => {
```

Comment:
Can we adjust `jsNumberForAddress` and `addressToNumber`  names to indicate which is meant for for eth and oasis?